backend/src/api/auth.ts

```diff
@@ -45,12 +45,18 @@ interface UserInfo {
 }
 
 const client = jwksClient({
-  jwksUri: `https://cognito-idp.us-east-1.amazonaws.com/${process.env.REACT_APP_USER_POOL_ID}/.well-known/jwks.json`
+  jwksUri: `https://cognito-idp.us-east-1.amazonaws.com/${process.env.REACT_APP_USER_POOL_ID}/.well-known/jwks.json`,
+  timeout: 30000 // Defaults to 30s
```

Comment:
Do we need this line?

Reply:
From my tests seems that we do, even though there is a default with the same value. I can try again to eliminate.